src/app/pages/my-observations/components/sent-list/sent-list.component.ts

```diff
@@ -52,7 +52,10 @@ export class SentListComponent implements OnInit {
         filter((diff) => diff < 0), //only fetch observations when num drafts decrease
         debounceTime(500) //wait a bit in case multiple observations were shipped
       )
-      .subscribe(() => this.initRegistrationSubscription());
+      .subscribe(() => {
+        this.initRegistrationSubscription();
+        this.observationService.forceUpdateObservationsForCurrentGeoHazard();
```

Comment:
@jorgkv Are there a better place for this line? It's purpose is to update the observations on the map after submission.

Reply:
There is probably already functionality in place for this, that broke when we updated the API, but I haven't been able to find it.

Reply:
Skal sies at dette ikke bare påvirker kartet. Det påvirker også blant annet listvyen og alt annet som er interessert i hva for observasjoner som ligger på Internett. Så skal den være på et bestemt sted er det mest rimelig at det er koblet til koden som håndterer synkronisering.

Reply:
Tenker fiksen bør ligge et annet sted. Hvis vi feks endrer slik at man ikke kommer til sent-list.component etter innsending så vil jo ikke dette ha noen effekt lenger i forhold til bugen vi prøver å løse?